pallets/nfts/src/weights.rs

```diff
@@ -2,13 +2,13 @@
 //! Autogenerated weights for `pallet_nfts`
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 40.0.0
-//! DATE: 2024-12-12, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2024-12-22, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! WORST CASE MAP SIZE: `1000000`
 //! HOSTNAME: `R0GUE`, CPU: `<UNKNOWN>`
 //! WASM-EXECUTION: `Compiled`, CHAIN: `Some("dev")`, DB CACHE: `1024`
 
 // Executed Command:
-// ./target/release/pop-node
+// ./target/production/pop-node
```

Comment:
DQ: I build with production mode instead of release, does it matter?

Reply:
No, we will need to benchmark using the production profile eventually. The only consideration here is that a production build takes longer, so release is fine for ongoing dev.